client.go

```diff
@@ -126,7 +127,8 @@ func (c Client) RawSecret(name string) (data []byte, ok bool) {
 		c.Warnf("Secret %v not found", name)
 		return nil, false
 	default:
-		c.Errorf("Bad response code getting secret %v: (status=%v, msg='%v')", name, resp.StatusCode, data)
+		msg := strings.Join(strings.Split(string(data), "\n"), " ")
```

Comment:
It does if it's one of the "usual" error messages and such from the keywhiz server.  We could escape it somehow so it's more readable but also not putting too much junk in logs.